book/actix-plugin.md

```diff
@@ -156,6 +156,55 @@ curl http://localhost:8080/api/spec
 
 Similarly, if we were to use other extractors like `web::Query<T>`, `web::Form<T>` or `web::Path`, the plugin will emit the corresponding specification as expected.
 
+#### Manually defining additional response codes
+
+There is a macro `api_v2_errors` which helps to manually add responses other than 200. Here is an example.
+We define an enum that represents the error. To use it in actix-web it needs to implement `Display` and `ResponseError`.
+There is no need to impl Display by hand for such simple enums - we can use `enum-display-derive` lib.
```

Comment:
Let's just mention that we offer a macro which is useful for defining error status codes? I don't want to mention any specifics like implementing `ResponseError` on enums or using `enum-display-derive` here. We can assume that our users are already using actix and are just looking for automatic spec generation.